martin/src/pg/builder.rs

```diff
@@ -126,10 +126,10 @@ impl PgBuilder {
         let mut pending = Vec::new();
         for (id, cfg_inf) in &self.tables {
             // TODO: move this validation to serde somehow?
-            if let Some(extent) = cfg_inf.extent {
-                if extent == 0 {
-                    return Err(InvalidTableExtent(id.to_string(), cfg_inf.format_id()));
-                }
+            if let Some(extent) = cfg_inf.extent
+                && extent == 0
+            {
```

Comment:
I think the following would be identical

```suggestion
            if cfg_inf.extent == Some(0) {
```